hazelcast/src/main/java/com/hazelcast/cluster/impl/ClusterServiceImpl.java

```diff
@@ -1060,6 +1074,44 @@ public void updateMembers(Collection<MemberInfo> members) {
         }
     }
 
+    private boolean shouldProcessMemberUpdate(Map<Address, MemberImpl> currentMembers,
+                                              Collection<MemberInfo> newMemberInfos) {
+
+        int currentMembersSize = currentMembers.size();
+        int newMembersSize = newMemberInfos.size();
+
+        if (currentMembersSize > newMembersSize) {
+            logger.severe("Received an older member update, no need to process...");
```

Comment:
Don't you want to send the trigger message here? I think that's what we discussed.

Also I wouldn't use the severe log-level. I don't see anything severe on this - given the current design it's something we can totally expect.